app/src/main/java/com/asfoundation/wallet/ui/wallets/WalletsViewHolder.kt

```diff
@@ -8,18 +8,27 @@ import com.asf.wallet.R
 import com.asfoundation.wallet.util.CurrencyFormatUtils
 import io.reactivex.subjects.PublishSubject
 import kotlinx.android.synthetic.main.other_wallet_card.view.*
+import kotlinx.android.synthetic.main.other_wallet_card.view.wallet_balance
+import kotlinx.android.synthetic.main.wallet_rounded_outlined_card.view.*
 
 class WalletsViewHolder(private val context: Context, itemView: View,
                         private val uiEventListener: PublishSubject<String>,
-                        private val currencyFormatUtils: CurrencyFormatUtils) :
+                        private val currencyFormatUtils: CurrencyFormatUtils,
+                        private val walletsViewType: WalletsViewType) :
     RecyclerView.ViewHolder(itemView) {
 
   @SuppressLint("SetTextI18n")
   fun bind(item: WalletBalance) {
-    itemView.inactive_wallet_address.text = item.walletAddress
-    itemView.wallet_balance.text = context.getString(
-        R.string.wallets_2nd_view_balance_title) + " " + item.balance.symbol + currencyFormatUtils.formatCurrency(
-        item.balance.amount)
+    if (walletsViewType == WalletsViewType.BALANCE) {
+      itemView.inactive_wallet_address.text = item.walletAddress
+      itemView.wallet_balance.text = context.getString(
```

Comment:
Can't we create a string with placeholders so we don't have concatenate multiple strings?